defaultModel/src/main/scala/pl/touk/nussknacker/defaultmodel/DefaultConfigCreator.scala

```diff
@@ -1,9 +1,9 @@
-package pl.touk.nussknacker.genericmodel
+package pl.touk.nussknacker.defaultmodel
 
 import pl.touk.nussknacker.engine.api.process._
 import pl.touk.nussknacker.engine.util.process.EmptyProcessConfigCreator
 
-class GenericConfigCreator extends EmptyProcessConfigCreator {
+class DefaultConfigCreator extends EmptyProcessConfigCreator {
```

Comment:
WDYT about other generic* -> default* renames like: `GenericMigrattions`, `extractedTypes/genericCreator.json `?

Reply:
:+1: I renamed what I managed to find...

Reply:
thanks